SimplenoteUITests/AllNotes.swift

```diff
@@ -5,29 +5,35 @@ class AllNotes {
     class func openNote(noteName: String) {
         app.tables.cells[noteName].tap()
     }
-    
+
```

Comment:
🎉 for removing trailing whitespaces. 👍 

Totally useless for how the code work, but sticklers like me really appreciate 🤣 🙇‍♂️